projects/netgrif-components-core/src/lib/panel/abstract/panel-with-immediate-data.ts

```diff
@@ -15,12 +15,16 @@ import {OverflowService} from '../../header/services/overflow.service';
 export abstract class AbstractPanelWithImmediateDataComponent extends AbstractPanelWithHeaderBindingComponent implements OnDestroy {
     protected constructor(protected _translate: TranslateService,
                           protected _currencyPipe: CurrencyPipe,
-                          protected _overflowService: OverflowService) {
+                          @Optional() protected _overflowService: OverflowService) {
         super();
     }
 
     get overflowMode(): boolean {
+        if(this._overflowService !== null){
```

Comment:
Same as above.

Cannot be simplified to `!!this._overflowService`?

Reply:
Done